### PR TITLE
 Switch to the built-in version of ClientWebSocket for Full .NET Framework

### PR DESCRIPTION
**Pull request by dstuckims**
…

## Description
Remove the clone/copy of the .NET 4.6 ClientWebSocket class.
Addresses: https://github.com/Azure/azure-relay-dotnet/issues/34

This checklist is used to make sure that common guidelines for a pull request are followed.
- [x] **I have read the [contribution guidelines](./CONTRIBUTING.md).**  (The link to CONTRIBUTING.md is broken, I believe).
- [X] Title of the pull request is clear and informative.
- [X] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR.
- [X] The pull request does not introduce breaking changes (unless a major version change occurs in the assembly and module).
- [X] If applicable, the public code is properly documented.
- [X] Pull request includes test coverage for the included changes.
- [X] The code builds without any errors.

### CODE REVIEW

File: src/Microsoft.Azure.Relay/HybridConnectionClient.cs [hunk 1]
```diff
@@ -178,16 +181,6 @@ public HybridConnectionClient(string connectionString, string path)
...
 
-#if NET45 // TODO: Flow Response Headers in NETSTANDARD ClientWebSocket
-                var trackingId = webSocket.ResponseHeaders[TrackingContext.TrackingIdName];
-                if (!string.IsNullOrEmpty(trackingId))
-                {
-                    // Update to the flown trackingId (which has _GX suffix)
-                    trackingContext = TrackingContext.Create(trackingId, trackingContext.SubsystemId);
-                    traceSource = nameof(HybridConnectionClient) + "(" + trackingContext.ToString() + ")";
-                }
-#endif
```
Comment (dstuckims) [inline src/Microsoft.Azure.Relay/HybridConnectionClient.cs]: This code (now being removed) was only possible due to custom changes I had made in our copy of the .NET 4.6 ClientWebSocket class.  Reading the HTTP Response headers is not possible with the built-in ClientWebSocket.

Comment (jtaubensee) [inline src/Microsoft.Azure.Relay/HybridConnectionClient.cs]: Will we have to update the recent unit test changes to account for this?

Comment (dstuckims) [inline src/Microsoft.Azure.Relay/HybridConnectionClient.cs]: This code was merely grabbing the returned "TrackingId" HTTP header in order to have a tracking ID with _GXX suffix.

Comment (dstuckims) [inline src/Microsoft.Azure.Relay/HybridConnectionClient.cs]: All the unit tests pass (with the WriteLargeData one still failing sometimes, but that's a separate issue I hope to address in the next few days).

**Pull request merged**